doc/port-config-refactor/port-config-refactor-design.md

```diff
@@ -0,0 +1,48 @@
+# SONiC Port Configuration Refactor Design #
+
```

Comment:
https://github.com/Azure/sonic-platform-common/pull/72   check this PR, not quite sure whether it's already covered some change?

Reply:
I don't understand that PR, it said it support platform.json, but I don't see any code parse platform.json

Reply:
then we ignor, I just thought maybe the also made some change.